README.md

```diff
@@ -66,4 +74,4 @@ that this can take a minute through the entire suite.
 
 
 
-  
\ No newline at end of file
+  
```

Comment:
Nit: can you remove this space, and maybe a few more before it. That's mostly my mistake.